sockets/sockets.go

```diff
@@ -35,3 +41,18 @@ func ConfigureTransport(tr *http.Transport, proto, addr string) error {
 	}
 	return nil
 }
+
+func configureUnixTransport(tr *http.Transport, proto, addr string) error {
+	if len(addr) > maxUnixSocketPathSize {
+		return fmt.Errorf("Unix socket path %q is too long", addr)
```

Comment:
```suggestion
		return fmt.Errorf("unix socket path %q is too long", addr)
```

Error strings should be lowercase (and this was changed in this PR)